src/main/java/dev/blachut/svelte/lang/codeInsight/SvelteTagProvider.kt

```diff
@@ -70,20 +74,26 @@ class SvelteTagProvider : XmlElementDescriptorProvider, XmlTagNameProvider {
             elements.addAll(svelteNamespacedTagLookupElements)
         }
 
-        val file = tag.containingFile
-        val visitor = SvelteScriptVisitor()
-        file.accept(visitor)
-        val jsElement = visitor.jsElement ?: return
+        val svelteFiles = FileTypeIndex.getFiles(SvelteFileType.INSTANCE, GlobalSearchScope.allScope(tag.project))
+        val reachableComponents = svelteFiles.map {
+            val componentName = it.nameWithoutExtension
+            val componentProps = ComponentPropsProvider().getComponentProps(it, tag.project)
 
-        val importVisitor = ImportVisitor()
-        jsElement.accept(importVisitor)
-        val items = importVisitor.components.map {
-            val lookupElement = LookupElementBuilder.create(it).withIcon(SvelteIcons.FILE)
+            val lookupObject = mapOf("file" to it, "props" to componentProps)
```

Comment:
Better to use data class
```
data class ComponentLookupObject(val file: VirtualFile, val props: List<String>)
```
or something like that

Reply:
Seems like that's the correct way of doing it, I will modify it then